frontend/common/types/requests.ts

```diff
@@ -138,18 +170,37 @@ export type Req = {
     is_archived?: boolean
   }
   getProjectFlag: { project: string; id: string }
-  getRolesPermissionUsers: { organisation_id: string; role_id: string }
+  getRolesPermissionUsers: { organisation_id: number; role_id: number }
   deleteRolesPermissionUsers: {
-    organisation_id: string
-    role_id: string
-    user_id: string
+    organisation_id: number
+    role_id: number
+    user_id: number
     level?: string
   }
-  createRolesPermissionUsers: { organisation_id: string; role_id: string }
-  getRolePermissionGroup: { id: string }
-  updateRolePermissionGroup: { id: string }
-  deleteRolePermissionGroup: { id: string }
-  createRolePermissionGroup: { organisation_id: string; role_id: string }
+  createRolesPermissionUsers: {
+    data: {
+      user: number
+    }
+    organisation_id: number
+    role_id: number
+  }
+  getRolePermissionGroup: {
+    organisation_id: number
+    role_id: number
+  }
+  updateRolePermissionGroup: { id: number; role_id: number }
+  deleteRolePermissionGroup: {
+    group_id: number
+    organisation_id: number
+    role_id: number
+  }
+  createRolePermissionGroup: {
+    data: {
+      group: number
+    }
```

Comment:
Should this be `group_id:`?

Reply:
No, the endpoint organisations/${query.organisation_id}/roles/${query.role_id}/groups takes group:number